craft_application/commands/base.py

```diff
@@ -53,13 +53,7 @@ class AppCommand(BaseCommand):
     always_load_project: bool = False
     """The project is also loaded in non-managed mode."""
 
-    def __init__(self, config: dict[str, Any] | None) -> None:
```

Comment:
question: I understand this can be removed because `app_config` from `Application` should now never be `None`. But I also understand `Application.run()` was refactored and `_run_inner()` extracted so it can be overridden by applications if necessary. In this case, applications will be able to instantiate an `AppCommand` without providing a `app_config`. Then, should we still check `if config is None`?

Reply:
updated